kloppy/infra/serializers/event/wyscout/serializer.py

```diff
@@ -0,0 +1,370 @@
+                            **shot_event_args, **generic_event_args
+                        )
+                    ]
+                elif raw_event["eventName"] == 8:
+                    pass_event_args = _parse_pass(raw_event, next_event)
+                    event = [
+                        PassEvent.create(
+                            **pass_event_args, **generic_event_args
+                        )
+                    ]
+                elif raw_event["eventName"] == 2:
+                    foul_event_args = _parse_foul(raw_event)
+                    event = [
+                        FoulCommittedEvent.create(
+                            **foul_event_args, **generic_event_args
+                        )
+                    ]
+                    if any(
+                        (_has_tag(raw_event, t) for t in [1701, 1702, 1703])
+                    ):
+                        card_event_args = _parse_card(raw_event)
+                        event.append(
+                            CardEvent.create(
+                                **card_event_args, **generic_event_args
+                            )
+                        )
+                elif raw_event["eventName"] == 5:
+                    ball_out_event_args = _parse_ball_out(raw_event)
+                    event = [
+                        BallOutEvent.create(
+                            **ball_out_event_args, **generic_event_args
+                        )
+                    ]
+                elif raw_event["eventName"] == 3:
+                    set_piece_event_args = _parse_set_piece(
+                        raw_event, next_event
+                    )
+                    if raw_event["subEventName"] in [30, 31, 32, 34, 36]:
+                        event = [
+                            PassEvent.create(
+                                **set_piece_event_args, **generic_event_args
+                            )
+                        ]
+                    elif raw_event["subEventName"] in [33, 35]:
+                        event = [
+                            ShotEvent.create(
+                                **set_piece_event_args, **generic_event_args
+                            )
+                        ]
+                    else:
+                        print(raw_event["subEventName"])
+                elif raw_event["eventName"] == 7:
+                    recovery_event_args = _parse_recovery(raw_event)
+                    event = [
+                        RecoveryEvent.create(
+                            **recovery_event_args, **generic_event_args
+                        )
+                    ]
+                elif raw_event["eventName"] == 1:
+                    takeon_event_args = _parse_takeon(raw_event)
+                    event = [
+                        TakeOnEvent.create(
+                            **takeon_event_args, **generic_event_args
+                        )
+                    ]
+
```

Comment:
Please add another case to collect all the GenericEvents (the ones we don’t recognize)

Reply:
Will need to add.